mapbox_gl_web/ios/mapbox_gl_web.podspec

```diff
@@ -0,0 +1,18 @@
+
+Pod::Spec.new do |s|
+    s.name             = 'mapbox_gl_web'
+    s.version          = '0.1.0'
+    s.summary          = 'No-op implementation of mapbox_gl_web web plugin to avoid build issues on iOS'
```

Comment:
Out of curiosity, what issues did you run into.

Reply:
this is a Flutter issue, all web version of packages have this file.. for example [shared_preferences](https://github.com/flutter/plugins/blob/master/packages/shared_preferences/shared_preferences_web/ios/shared_preferences_web.podspec)